curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/access-the-json-data-from-an-api.english.md

```diff
@@ -8,14 +8,25 @@ challengeType: 6
 <section id='description'>
 In the previous challenge, you saw how to get JSON data from the freeCodeCamp Cat Photo API.
 Now you'll take a closer look at the returned data to better understand the JSON format. Recall some notation in JavaScript:
-<blockquote>[ ] -> Square brackets represent an array<br>{ } -> Curly brackets represent an object<br>" " -> Double quotes represent a string. They are also used for key names in JSON</blockquote>
+
+```js
```

Comment:
```suggestion
```
```

Reply:
I think it was decided to stick with the blockquotes on non-code - I'm fine with the unlabeled backticks - I'll leave it up to you guys - Other than this, it all looks good 🎉

Reply:
Done.